mantidproject/mslice
# Plot interactivity - cuts

**Not to be merged until after #187**

Features of a cut plot can now be changed by clicking on them (issue #180). This includes labels,  axis ranges, and the actual lines.

**To test:**
-  Plot a cut
- Click a feature (axis labels or ranges, lines)
- Change settings in the dialog
- Click OK
- Check that the plot is updated with new settings
- Repeat for each feature

Fixes #180
